pkgs/development/python-modules/zconfig/default.nix

```
@@ -16,13 +17,16 @@ buildPythonPackage rec {
     sha256 = "0s7aycxna07a04b4rswbkj4y5qh3gxy2mcsqb9dmy0iimj9f9550";
   };
 
-  patches = stdenv.lib.optional stdenv.hostPlatform.isMusl ./remove-setlocale-test.patch;
+  patches = [
+    (fetchpatch {
```

generally I like to state why the patch is being included, and when it can be removed
```suggestion
    # fixes 3.8+ logger validation issues, has been merged into master, remove next bump
    (fetchpatch {
```